### PR TITLE
Fix: Clean up some more automate styling

### PR DESCRIPTION
**Pull request by Mikehrn**
Bit more cleaning up of automate UI. Again, can be improved further, but just a start to align things a bit more with the other styles

Screenshot 2024-11-25 at 20 29 56

Screenshot 2024-11-25 at 20 29 41

Screenshot 2024-11-25 at 20 27 41

Screenshot 2024-11-25 at 20 15 47

### CODE REVIEW

File: packages/frontend-2/components/automate/function/page/ParametersDialog.vue [hunk 1]
```diff
@@ -1,19 +1,19 @@
...
-          border-t
-          border-b
-        >
+        <LayoutDialogSection title="Parameter schema" border-t border-b>
           <FormTextArea
             name="actionYaml"
             readonly
             :model-value="JSON.stringify(finalParams, null, 2)"
-            class="text-sm text-primary"
+            textarea-classes="!bg-foundation !border border-outline-2 p-2 rounded-lg font-mono !font-body-2xs"
```
Comment (benjaminvo) [inline packages/frontend-2/components/automate/function/page/ParametersDialog.vue:16]: I haven't seen `textarea-classes` before 🤔. Is this a real prop?

**Review by benjaminvo — APPROVED** [review]
All looks very good! One comment – but I've approved anyway in case it is as it should be. Otherwise fix and I'll reapprove.

File: packages/frontend-2/components/automate/function/page/ParametersDialog.vue (hunk 1 above)
Comment (Mikehrn) [reply packages/frontend-2/components/automate/function/page/ParametersDialog.vue:16]: Yup it is, it's used only once or twice I think, but inside `FormTextArea` is a nested `textarea` element, this prop allows you to set the classes on that element :)

Comment (benjaminvo) [reply packages/frontend-2/components/automate/function/page/ParametersDialog.vue:16]: Ah!

**Pull request merged**